src/api/stop-places/index.ts

```diff
@@ -36,4 +38,18 @@ export default (server: Hapi.Server) => (service: IStopPlacesService) => {
       return (await service.getStopPlaceConnections(query, h.request)).unwrap();
     },
   });
+  server.route({
+    method: 'GET',
+    path: '/bff/v2/stop-places/parent',
```

Comment:
Since it's just the ID and nothing more, maybe that can be part of the endpoint name? 🤔

```suggestion
    path: '/bff/v2/stop-places/parent-id',
```